CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Breaking changes:
     - `debug.restart` renamed to `workbench.action.debug.restart`
 - [preferences] renamed overridenPreferenceName to overriddenPreferenceName
 - [task] `cwd`, which used to be defined directly under `Task`, is moved into `Task.options` object
+- [preferences] removed constructor from the `FolderPreferenceProvider` class
```

Comment:
Was there a problem with the constructor injection? 😊

Reply:
there is nothing wrong with the contructor injection. I removed the them simply because none of the injected is referenced in the class. Also, in our coding standard / guidelines, we should try using property injection https://github.com/theia-ide/theia/wiki/Coding-Guidelines#property-injection over the constructor injection. 

If you are not happy with this change I can definitely remove it - having unused injected variables wouldn't cause any issues :)